ormconfig.js

```diff
@@ -20,8 +18,8 @@ module.exports = [
   {
     type: 'postgres',
     url: process.env.DATABASE_URL,
-    ssl: sslConfig.ssl,
-    extra: sslConfig.extra,
+    ssl: prodSSLConfig.ssl,
```

Comment:
why is this being done? are we not doing sslConfig based on node_env anymore?

Reply:
Ah crap, this is because I was testing on dev db :)))))